### ISSUE TITLE
Reviewing GitHub 101

### ISSUE THREAD

**Issue by beardofedu** [github/training-kit#568]
## Overview
**TL;DR**

Reviewing the GitHub 101 and identifying any new features, bug fixes, or general items that need eyes or a refresh.

- [x] References to Continue button should be written with a capital C to be consistent with button look / feel (Introduction to GitHub page)
- [x] Standardize **bolding** in content
- [x] Select the `Unlimited public repositories for free.` option. (Create a GitHub Account)
- [x] `awesome photo and a fun bio.` Change photo to profile picture. (Link to Help doc https://help.github.com/articles/setting-your-profile-picture/)
- [x] Change the wording for Step 2. (Join the Class Repository)
- [ ] gif is inaccurate due to new template (Join the Class Repository)
   - [ ] all gifs are inaccurate because they show Pulse / Graphs
- [x] Change `Navigate to the Code tab to find the README.` to include `repository README` or something (Explore the GitHub Repository)
- [x] Pulse and Graphs are referenced without the correct UI context (inside Insights) (Explore the GitHub repository)
- [x] Use GitHub Flow doesn't really have steps, seems to be an introduction to GH Flow and might be able to be moved to Create a Branch
- [x] Create a Branch needs bolding stuff
- [x] gif doesn't use the site that is linked for Manually Search (Find Your Coordinates) https://mynasadata.larc.nasa.gov/latitudelongitude-finder/
- [x] button text is Create pull request, (Create a Pull Request)
- [x] Reference 404 error, but that should be limited to the GitHub Pages courses and not this one. Opportunity to identify some of the common reasons a build fails. (Edit Files on GitHub)
- [x] Issue with image in Tell Me Why _Merge visual_ (Merge Your Pull Request)
- [x] Modify end para / Add `heart` to the We :heart: Feedback title (You Completed the GitHub Flow)

**Comment by crichID** [comment]
@beardofedu so many :heart: :heart:s for doing this! Thank you ...

**Comment by beardofedu** [comment]
While most of these edits are straightforward, I could use some opinions on how to deal with this:

> Use GitHub Flow doesn't really have steps, seems to be an introduction to GH Flow and might be able to be moved to Create a Branch

Basically, this topic is 1) really short 2) doesn't contain any steps. So, I can move the content to the Create a Branch topic **or** make it more informational and bring in a little more content. 

@github/services-training for 💭 s

**Comment by brianamarie** [comment]
> So, I can move the content to the Create a Branch topic or make it more informational and bring in a little more content.

I vote for adding more information and adding content. It's the main point of the entire course, so I think it's worth it. We could easily add [this video](https://www.youtube.com/watch?v=47E-jcuQz5c&t) as one resource.

**Comment by beardofedu** [comment]
> Reference 404 error, but that should be limited to the GitHub Pages courses and not this one. 

This was fixed by modifying content in GH 102 and 103 (@hectorsector). I simply created a new file titled `help-gh102-103-404.md` and included it in the `16-collaborate.md` file for each of those courses. 

This removed the 404 error content that was incorrectly displaying in the GH 101 course, but retains it in the 102 | 103 courses.

**Comment by brianamarie** [comment]
@beardofedu What are your thoughts on moving the gifs to the top of the page instead of in the "show me how"? 🤔

**Comment by beardofedu** [comment]
@brianamarie it seems like that is the direction we are going with all of the other courses so I am all for it.
